homepairs/homepairsUI/src/Routes/Routes.tsx

```diff
@@ -75,7 +76,7 @@ const innerStackConfig: any = {
 
 //these should be separated into different files for each Route (AccountProperties, Service Request, Account)
 const propertyStackConfig_PM = {
-  initialRouteName: 'AccountProperties',
+  initialRouteName: 'TenantProperties',
```

Comment:
why is PM config initially routing to tenant properties?

Reply:
I outright removed the propertyStack_PM and propertyStack_TENANT and simply put them in the same stack. I resolve them with a new function called ChooseMainPage() which all the fetchAccount actions call. We just default to TenantProperties page. However, this initial Route should never be called if we are making sure to always calling new function I defined. 

We should rename the configuration however. I can update that right now.